src/CaesHelp/ClientApp/src/components/Ticket.tsx

```diff
@@ -250,11 +263,11 @@ export default class Ticket extends React.Component<
 
   public render() {
     const programmingSupportTitle =
-      '<b>Programming Support:</b> (Scott Kirkland, Ken Taylor, Jason Sylvestre, Spruce Weber-Milne)';
+      '<b>Programming Support:</b> (Scott, Ken, Jason, Spruce)';
```

Comment:
should i be in here? :)

Reply:
@laholstege Only if you have access to Service Now to answer help tickets.

Reply:
Only if you have access to Service Now to answer help tickets

Reply:
cool, i don't and i'm happy with that